.sphinx/_templates/header.html

```diff
@@ -28,9 +27,20 @@
             <a href="{{ github_url }}" class="p-navigation__sub-link p-dropdown__link">GitHub</a>
           </li>
 
+        </ul> 
+
+      </li>
+      <li>
+        {% set contributors = get_contribs(github_url, github_folder, pagename, page_source_suffix) %}
```

Comment:
You should check here if `github_url`, `github_folder`, `pagename`, and `page_source_suffix` are actually set, and if any of them are missing, you can skip the whole section.

We don't require the GitHub URL to be set (for example, it could be empty if the files are in a private repository). And in this case, we cannot display the contributors.